Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -262,4 +262,28 @@ public function testNestedProperties()
             ),
         ));
     }
+
+    public function testCompressionConfig()
+    {
+        $configuration = $this->getConfigs(array(
+            'clients' => array(
+                'client_1' => array('compression' => true),
+                'client_2' => array('compression' => false),
+            ),
+        ));
+
+        $this->assertTrue($configuration['clients']['client_1']'compression']);
```

Comment:
Still typo here & below... There is `$configuration['clients']['client_1']'compression']` but should be `$configuration['clients']['client_1']['compression']`...